sdk/src/builder.rs

```diff
@@ -164,6 +164,53 @@ impl AssertionDefinition {
     }
 }
 
+#[derive(Debug, Default, Deserialize, Serialize, Clone, PartialEq)]
+#[cfg_attr(feature = "json_schema", derive(JsonSchema))]
+pub enum DigitalSourceType {
+    /// Media whose digital content is effectively empty, such as a blank canvas or zero-length video.
+    #[default]
+    Empty,
+    /// Data that is the result of algorithmically using a model derived from sampled content and data.
+    /// Differs from <http://cv.iptc.org/newscodes/digitalsourcetype/>trainedAlgorithmicMedia in that
+    /// the result isn’t a media type (e.g., image or video) but is a data format (e.g., CSV, pickle).
+    TrainedAlgorithmicData,
+}
+
+impl DigitalSourceType {
+    pub fn as_url(&self) -> &'static str {
+        match self {
+            Self::Empty => "http://c2pa.org/digitalsourcetype/empty",
+            Self::TrainedAlgorithmicData => {
+                "http://c2pa.org/digitalsourcetype/trainedAlgorithmicData"
+            }
+        }
+    }
+}
+
+/// Represents the type of builder flow being used.
+/// This determines how the builder will be used, such as creating a new asset, opening an existing asset,
+/// or updating an existing asset.
+#[derive(Debug, Default, Deserialize, Serialize, PartialEq, Clone)]
+#[cfg_attr(feature = "json_schema", derive(JsonSchema))]
+enum BuilderFlow {
+    // Represents an unspecified builder flow.
+    #[default]
+    None,
```

Comment:
How come you include a `None` variant rather than an `Option<BuilderFlow>`?

Reply:
I don't remember, I changed this to an Option